src/status_im/common/scroll_page/view.cljs

```diff
@@ -64,9 +64,7 @@
         [quo/page-nav
          (cond-> {:margin-top     (safe-area/get-top)
                   :type           :no-title
-                  :background     (if (= 1 (reanimated/get-shared-value opacity-animation))
-                                    :blur
-                                    :photo)
+                  :background     :blur
```

Comment:
I was trying to look in the designs if there is any instance where that scrollable page component uses top button with type `blur` but couldn't find any. cc @J-Son89

Reply:
ah, I believe it's like this when the animation is complete so you might have to check that on the communitys page for example.

Reply:
e.g https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=1733-127837&mode=design&t=IMyYUqCrxBlSuvzE-4